lib/imports.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const {getOriginal} = require('./originals-helper.js');
+
+/**
+ * Formats the information into the right format for import
+ *
+ * @param {string} identifier - the name of the method / function to get
+ * @param {string} type - the type of original of the identifier
+ * @param {string} mod - the name of the module to import from
+ *
+ * @return {string} the formatted import text
+ */
+function generateImportText(identifier, type, mod) {
+  const original = getOriginal(identifier, type, mod);
+  switch (type) {
+    case 'constructor':
+      return original;
+    case 'method':
+      return `${identifier} as ${original}`;
+    default:
+      throw new Error('not yet implemented');
+  }
+}
+
+/**
+ * Adds an import for the module to the list
+ *
+ * @param {string} mod - the name of the module to import from
+ * @param {string} identifier - the name of the method / function to get
+ * @param {string} type - the type of original of the identifier
+ * @param {Object} imports - the current list of imports
+ */
+function updateImports(mod, identifier, type, imports) {
```

Comment:
Nit: I think putting imports as the first argument would make more sense, since it's what you're updating on. `updateImports(imports, ... stuff to use in updating ...)` reads more naturally than `updateImports(... stuff ..., imports)`.